### PR TITLE
Refactor bicep parameters and outputs

### PR DESCRIPTION
**Pull request by abatallas**
This initial commit shows changes to the UI definition file's outputs and associated changes to the body. It currently does not pass the arm-ttk as changes need to be made to the Bicep files. This approach avoids the massive ccswConfig object as well as the parse(concat()) monstrosities.

### CODE REVIEW

**Comment by abatallas** [comment]
Note: This is still a WIP. Please stay tuned for more updates.

File: bicep/ccsw.bicep [hunk 1]
```diff
@@ -1,408 +1,293 @@
...
+
 param adminUsername string
 @secure()
 param adminPassword string
+//param adminKeyphrase string
 param adminSshPublicKey string
-param storedKey object = {}
-
-//cc vm parameters
+param storedKey object
```
Comment (ryanhamel) [inline bicep/ccsw.bicep:16]: Let's type storedKey as well

File: bicep/ccsw.bicep [hunk 2]
```diff
@@ -1,408 +1,293 @@
...
+    database_user: adminUsername
+    azure_environment: envNameToCloudMap[environment().name]
+    blob_storage_suffix: 'blob.${environment().suffixes.storage}' // blob.core.windows.net
   },
   {}
 )
 
-output trash object = trash_for_arm_ttk
 output branch string = branch
 output project_version string = project_version
```
Comment (ryanhamel) [inline bicep/ccsw.bicep:293]: Let's use camelCase for all variables in a separate commit

File: bicep/ccsw.bicep [hunk 3]
```diff
@@ -1,408 +1,293 @@
 targetScope = 'resourceGroup'
+//import {shared_filesystem_t, additional_filesystem_t, vnet_t, tags_t, slurmSettings_t, login_t, htc_t, hpc_t, htc_output_t} from './types.bicep'
```
[the remark this answers is not in the document]
Comment (abatallas) [reply bicep/ccsw.bicep:2]: Done.

File: bicep/ccsw.bicep (hunk 1 above)
Comment (abatallas) [reply bicep/ccsw.bicep:16]: Done.

**Review by ryanhamel — APPROVED** [review]

**Pull request merged**